demo/clusters/kind/scripts/common.sh

```diff
@@ -46,7 +46,9 @@ DRIVER_IMAGE_VERSION=$(from_versions_mk "VERSION")
 : ${KIND_CLUSTER_CONFIG_PATH:="${SCRIPTS_DIR}/kind-cluster-config.yaml"}
 
 # The derived name of the kind image to build
-: ${KIND_IMAGE_BASE_TAG:="v20230515-01914134-containerd_v1.7.1"}
-: ${KIND_IMAGE_BASE:="gcr.io/k8s-staging-kind/base:${KIND_IMAGE_BASE_TAG}"}
-: ${KIND_IMAGE:="kindest/node:${KIND_K8S_TAG}-${KIND_IMAGE_BASE_TAG}"}
+${NVIDIA_DRIVER_ROOT:+--set nvidiaDriverRoot=${NVIDIA_DRIVER_ROOT}}
```

Comment:
What does the `:+` yntax do exactly? Where is this variable used later on and why wasn't it needed in v1.27.1?

Reply:
`:+` only expands the section that follows it if the variable is defined.

For example:
```
$ export FOO=baz
$ echo "some/thing/possibly${FOO:+-${FOO}}"
some/thing/possibly-bar
$ unset FOO
$ echo "some/thing/possibly${FOO:+-${FOO}}"
some/thing/possibly
```

Reply:
Sorry, now I see the confusion. The `NVIDIA_DRIVER_ROOT` is *not* required. I had pasted the syntax here as a reminder while working on the usage of `:+` below but never removed it.

Removed now.

Reply:
It still seems to be there.

Reply:
Yes. I had forgotten to push.